descwl_shear_sims/sim.py

```diff
@@ -197,7 +197,6 @@ def make_sim(
 def make_exp(
```

Comment:
I'd move all of the galsim RNG instantiations to this function. This keeps the APIs the same and properly decouples the RNG from the rest of the code.

Reply:
Wouldn't it be better to have it declare at the same time as the numpy rng?

Reply:
Nope. The sim seed is set by the numpy RNG seed. We need to chain the galsim one to it via what I said.

Reply:
Ok I think I understand better.
So I should do something like [this](https://github.com/LSSTDESC/descwl-shear-sims/blob/02fd39dbe4c982c575193ea861bbaa51b99466f0/shear_meas_tests/test_shear_meas_corr_noise.py#L191) `rng.randint(low=1, high=2**29, size=ntrial)` ?

Reply:
Yeah just set the size to 1 and feed this as a seed to the GALSIM RNG